src/Uno.UI/UI/Xaml/PropertyMetadataDictionary.cs

```diff
@@ -20,17 +20,16 @@ internal class PropertyMetadataDictionary
 		/// This implementation of PropertyMetadataDictionary uses HashTable because it's generally faster than
 		/// Dictionary`ref/ref. 
 		/// </summary>
-		private readonly HashtableEx _table = new HashtableEx();
+		private readonly Hashtable _table = new Hashtable();
```

Comment:
This is created per dependency property instance. So there is a hashtable for each DP, and it's likely that all these instances live for the lifetime of the app.